pages/common/mlr.md

```diff
@@ -0,0 +1,27 @@
+# mlr
+
+> Miller is like `awk`, `sed`, `cut`, `join`, and `sort` for name-indexed data such as CSV, TSV, and tabular JSON.
+
+- Pretty-print a CSV file in a tabular format:
+
+`mlr --icsv --opprint cat {{example.csv}}`
+
+- Receive JSON data and pretty print the output:
+
+`echo '{"hello":"world"}' | mlr --ijson --opprint cat`
+
+- Sort alphabetically on field:
+
+`mlr --icsv --opprint sort -f {{field}} {{example.csv}}`
+
+- Sort in descending numerical order on field:
+
+`mlr --icsv --opprint sort -nr {{field}} {{example.csv}}`
+
+- Convert CSV to JSON, performing calculations and display those calculations:
+
+`mlr --icsv --ojson put '${{newField1}} = ${{oldFieldA}}/${{oldFieldB}}' {{example.csv}}`
+
+- Retrieve records from a MySQL database and format the output as vertical JSON:
```

Comment:
Sorry for one last comment, but can we show another example other than `mysql` ? Something like we did with `echo {"hello": "world"}`. It's just that with mysql, the example is becoming too long. We just want to show that it formats the output as vertical json.

Reply:
No apologies necessary, thank you for your guidance and your patience!